reference_code/verifiers/circom/solidity/.gitignore

```diff
@@ -0,0 +1,15 @@
+/build
```

Comment:
.gitignore this file

Reply:
addressed by https://github.com/Sindri-Labs/sindri-resources/pull/68/commits/46547a7e99eefaeb2f12f563d26072efc2121751